src/utils/wallet.js

```diff
@@ -600,9 +601,11 @@ class Wallet {
 
     async getBalance(accountId) {
         accountId = accountId || this.accountId
-
         const account = this.getAccount(accountId)
-        return await account.getAccountBalance()
+        let balance = await account.getAccountBalance()
+        balance.stateStaked = new BN(balance.stateStaked).add(new BN(MIN_BALANCE_FOR_GAS)).toString()
+        balance.available = new BN(balance.available).sub(new BN(MIN_BALANCE_FOR_GAS)).toString()
```

Comment:
@Patrick1904 looks good to me, but need to make sure that doesn't add twice with `account-with-lockup`  (when account has lockup).

Reply:
@vgrichina I just tested on staging and it works as intended for `account-with-lockup` too. Can you please test and confirm with a mainnet account too? Thank you